hash_practice/exercises.py

```diff
@@ -5,17 +5,56 @@ def grouped_anagrams(strings):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    if len(strings) == 0:
+        return []
+    if len(strings) == 1:
+        return [[strings[0]]]
+    list1 = [strings[0]]
+    listOfLists = [list1]

```

Comment:
These are rather generic names that more describe what the data type is rather than what it is for. You don't even need `list1` because you can just do `[[strings[0]]]`, and something like `return_value` or the like would be more informative than `listOfLists`.